release/goreleaser.opm.Dockerfile

```
@@ -2,7 +2,11 @@
 #   build opm images. See the configurations in .goreleaser.yaml
 #   and .github/workflows/release.yaml.
 
-FROM --platform=$BUILDPLATFORM ghcr.io/grpc-ecosystem/grpc-health-probe:v0.4.4 as grpc_health_probe
+FROM ubuntu as grpc_health_probe
+RUN apt update && \
+  apt install -y curl && \
+  curl -o /grpc_health_probe \
+  https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.4.5/grpc_health_probe-linux-$TARGETARCH
```

Does this need to be `chmod`-ed so that it's executable?
```suggestion
  https://github.com/grpc-ecosystem/grpc-health-probe/releases/download/v0.4.5/grpc_health_probe-linux-$TARGETARCH && \
  chmod 755 /grpc_health_probe
```

---

That's a good catch! we should make sure!